client/src/main/java/io/split/engine/splitter/Splitter.java

```diff
@@ -11,8 +12,10 @@
  * @author adil
  */
 public class Splitter {
+    private static final int ALGO_MURMUR = 1;
```

Comment:
murmur is actually #2. see ticket: https://splitio.atlassian.net/browse/SPLIT-2578